tests/smoke/features/shareFileJourney.feature

```diff
@@ -1,42 +1,102 @@
+  Alice shares the folder with file to Brian with the "editor" role.
+  I want to check that Brian can accept folder, download, move, copy and rename the shared file
+  Alice shares file with Brian
+  I want to check that Brian can open file in Mediaviewer, copy, download the file.
+  I also want to check that Alice can change the role to the share and can delete share
 
   Background:
     Given following users have been created
       | Alice |
       | Brian |
     And admin set the default folder for received shares to "Shares"
+    And admin disables auto accepting
 
   Scenario: Alice shares folder with file to Brian
     Given "Alice" has logged in
-    Then "Alice" opens the "files" app
-    Then "Alice" navigates to the files page
-    Then "Alice" creates following folders
-      | folder_to_shared           |
-      | folder_to_share1/subFolder |
+    When "Alice" opens the "files" app
+    And "Alice" navigates to the files page
+    And "Alice" creates following folders
+      | folder_to_shared |
     And "Alice" uploads following resources
       | resource  | to               |
       | lorem.txt | folder_to_shared |
-    Then "Alice" moves following resources
-      | resource                             | to                         |
-      | folder_to_shared/lorem.txt           | folder_to_share1           |
-      | folder_to_share1/lorem.txt           | folder_to_share1/subFolder |
-      | folder_to_share1/subFolder/lorem.txt | folder_to_shared           |
-    Then "Alice" copies following resources
-      | resource                   | to                         |
-      | folder_to_shared/lorem.txt | folder_to_share1           |
-      | folder_to_share1/lorem.txt | folder_to_share1/subFolder |
-    Then "Alice" shares following resources
+    Then "Alice" ensures that the following resources exist
+      | folder_to_shared/lorem.txt |
+    And "Alice" shares following resources
       | resource         | user  | role   |
       | folder_to_shared | Brian | editor |
     Given "Brian" has logged in
-    Then "Brian" opens the "files" app
-    Then "Brian" accepts following resources
+    When "Brian" opens the "files" app
+    And "Brian" accepts following resources
       | folder_to_shared |
-    Then "Brian" renames following resource
-      | resource                          | newName      |
-      | Shares/folder_to_shared/lorem.txt | loremNew.txt |
+    And "Brian" renames following resource
+      | resource                          | as            |
+      | Shares/folder_to_shared/lorem.txt | lorem_new.txt |
     And "Brian" uploads following resources
       | resource   | to                      |
       | simple.pdf | Shares/folder_to_shared |
+    And "Brian" copies following resources
+      | resource                | to       |
+      | Shares/folder_to_shared | All files |
+    When "Alice" opens the "files" app
+    Then "Alice" ensures that the following resources exist
```

Comment:
We could change the words of this `Then` step so that it says what "should" be the case. Usually we write `Then` steps in a way that contains the word "should" (they "assert" something about the system/browser/UI-under-test)

But I will make suggestions about that separately, after this PR is merged. There are other small grammar things in the test step words, like `When "Brian" copies following resources` can be `When "Brian" copies the following resources` - that will help my English brain to read the scenario more easily - my brain currently does a small "interrupt" each time it notices that a small word is missing :)